### DISCUSSION TITLE
Solve "autofit guest" problem with analyst workstation (VMware)

### DISCUSSION THREAD

**Discussion by gebhard73** [Security-Onion-Solutions/securityonion#5478] [Show and tell] [answered]
Hi,

using analyst workstation in VMware workstation I ran into the problem that "autofit guest" wasn't working as expected. It turned out that the desktop VM tools were missing. This should do the magic:
`sudo yum install open-vm-tools-desktop` 

HTH.

**Comment by gebhard73** [comment, marked as answer]
answer above...

`sudo yum install open-vm-tools-desktop`

**Reply by dougburks** [reply]
Added to documentation:
https://docs.securityonion.net/en/2.3/vmware.html#tools
